addons/languages/coptic/pack/src/main/res/values-fr/strings.xml

```diff
@@ -1,5 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
     <string name="keyboard_name">ⲕⲏⲙⲉ</string>
+    <string name="keyboard_name_scriptorium">ⲕⲏⲙⲉ Scriptorium</string>
+    <string name="keyboard_name_swift">ⲕⲏⲙⲉ Swift</string>
```

Comment:
I take it back. I still like the ⲕⲏⲙⲉ, at least in the noto font.

One idea for shortening this would be to use **ⲕⲏⲙⲉ ͼ**, replacing _scriptorium_ with GREEK DOTTED LUNATE SIGMA SYMBOL, the editorial mark the project uses for a logo.

Swift might be something like **ⲕⲏⲙⲉ ⲙ̅ⲁⲥⲁⲓ**.